AES v0.9.0-rc.2 suddenly fails to build

**Minimal complete example:**

```rs
use aes::cipher::KeyInit;

fn main() {
    let key = b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00";
    aes::Aes256Enc::new_from_slice(key);
}
```

```
[package]
name = "aes-test"
version = "0.1.0"
edition = "2024"

[dependencies]
aes = "=0.9.0-rc.2"
```

----

**Result:**
```
$ cargo build
   Compiling aes v0.9.0-rc.2
error[E0432]: unresolved import `cipher::crypto_common::WeakKeyError`
  --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:21:5
   |
21 |     crypto_common::WeakKeyError,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WeakKeyError` in the root

error[E0432]: unresolved import `cipher::crypto_common::WeakKeyError`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/lib.rs:152:41
    |
152 | use cipher::{array::Array, consts::U16, crypto_common::WeakKeyError};
    |                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WeakKeyError` in the root

error[E0432]: unresolved import `cipher::crypto_common::WeakKeyError`
  --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:21:5
   |
21 |     crypto_common::WeakKeyError,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WeakKeyError` in the root
   |
   = help: consider importing this unresolved item through its public re-export instead:
           crate::WeakKeyError

error[E0432]: unresolved import `cipher::crypto_common::WeakKeyError`
 --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:9:5
  |
9 |     crypto_common::WeakKeyError,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WeakKeyError` in the root
  |
  = help: consider importing this unresolved item through its public re-export instead:
          crate::WeakKeyError

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:61:13
    |
 61 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
 62 | |                 crate::weak_key_test(&key.0)
 63 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
287 | / define_aes_impl!(
288 | |     Aes128,
289 | |     Aes128Enc,
290 | |     Aes128Dec,
...   |
298 | |     "AES-128",
299 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:138:13
    |
138 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
139 | |                 crate::weak_key_test(&key.0)
140 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
287 | / define_aes_impl!(
288 | |     Aes128,
289 | |     Aes128Enc,
290 | |     Aes128Dec,
...   |
298 | |     "AES-128",
299 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:187:13
    |
187 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
188 | |                 crate::weak_key_test(&key.0)
189 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
287 | / define_aes_impl!(
288 | |     Aes128,
289 | |     Aes128Enc,
290 | |     Aes128Dec,
...   |
298 | |     "AES-128",
299 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:61:13
    |
 61 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
 62 | |                 crate::weak_key_test(&key.0)
 63 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
301 | / define_aes_impl!(
302 | |     Aes192,
303 | |     Aes192Enc,
304 | |     Aes192Dec,
...   |
312 | |     "AES-192",
313 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:138:13
    |
138 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
139 | |                 crate::weak_key_test(&key.0)
140 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
301 | / define_aes_impl!(
302 | |     Aes192,
303 | |     Aes192Enc,
304 | |     Aes192Dec,
...   |
312 | |     "AES-192",
313 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:187:13
    |
187 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
188 | |                 crate::weak_key_test(&key.0)
189 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
301 | / define_aes_impl!(
302 | |     Aes192,
303 | |     Aes192Enc,
304 | |     Aes192Dec,
...   |
312 | |     "AES-192",
313 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:61:13
    |
 61 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
 62 | |                 crate::weak_key_test(&key.0)
 63 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
315 | / define_aes_impl!(
316 | |     Aes256,
317 | |     Aes256Enc,
318 | |     Aes256Dec,
...   |
326 | |     "AES-256",
327 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:138:13
    |
138 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
139 | |                 crate::weak_key_test(&key.0)
140 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
315 | / define_aes_impl!(
316 | |     Aes256,
317 | |     Aes256Enc,
318 | |     Aes256Dec,
...   |
326 | |     "AES-256",
327 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/soft.rs:187:13
    |
187 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
188 | |                 crate::weak_key_test(&key.0)
189 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
315 | / define_aes_impl!(
316 | |     Aes256,
317 | |     Aes256Enc,
318 | |     Aes256Dec,
...   |
326 | |     "AES-256",
327 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:213:13
    |
213 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
214 | |                 crate::weak_key_test(&key.0)
215 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
697 | / define_aes_impl!(
698 | |     Aes128,
699 | |     Aes128Enc,
700 | |     Aes128Dec,
...   |
705 | |     "AES-128",
706 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:301:13
    |
301 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
302 | |                 crate::weak_key_test(&key.0)
303 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
697 | / define_aes_impl!(
698 | |     Aes128,
699 | |     Aes128Enc,
700 | |     Aes128Dec,
...   |
705 | |     "AES-128",
706 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:376:13
    |
376 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
377 | |                 crate::weak_key_test(&key.0)
378 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
697 | / define_aes_impl!(
698 | |     Aes128,
699 | |     Aes128Enc,
700 | |     Aes128Dec,
...   |
705 | |     "AES-128",
706 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:213:13
    |
213 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
214 | |                 crate::weak_key_test(&key.0)
215 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
708 | / define_aes_impl!(
709 | |     Aes192,
710 | |     Aes192Enc,
711 | |     Aes192Dec,
...   |
716 | |     "AES-192",
717 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:301:13
    |
301 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
302 | |                 crate::weak_key_test(&key.0)
303 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
708 | / define_aes_impl!(
709 | |     Aes192,
710 | |     Aes192Enc,
711 | |     Aes192Dec,
...   |
716 | |     "AES-192",
717 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:376:13
    |
376 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
377 | |                 crate::weak_key_test(&key.0)
378 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
708 | / define_aes_impl!(
709 | |     Aes192,
710 | |     Aes192Enc,
711 | |     Aes192Dec,
...   |
716 | |     "AES-192",
717 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:213:13
    |
213 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
214 | |                 crate::weak_key_test(&key.0)
215 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
719 | / define_aes_impl!(
720 | |     Aes256,
721 | |     Aes256Enc,
722 | |     Aes256Dec,
...   |
727 | |     "AES-256",
728 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:301:13
    |
301 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
302 | |                 crate::weak_key_test(&key.0)
303 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
719 | / define_aes_impl!(
720 | |     Aes256,
721 | |     Aes256Enc,
722 | |     Aes256Dec,
...   |
727 | |     "AES-256",
728 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/x86.rs:376:13
    |
376 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
377 | |                 crate::weak_key_test(&key.0)
378 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
719 | / define_aes_impl!(
720 | |     Aes256,
721 | |     Aes256Enc,
722 | |     Aes256Dec,
...   |
727 | |     "AES-256",
728 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:105:13
    |
105 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
106 | |                 crate::weak_key_test(&key.0)
107 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
396 | / define_aes_impl!(
397 | |     name = Aes128,
398 | |     name_enc = Aes128Enc,
399 | |     name_dec = Aes128Dec,
...   |
402 | |     doc = "AES-128",
403 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:209:13
    |
209 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
210 | |                 crate::weak_key_test(&key.0)
211 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
396 | / define_aes_impl!(
397 | |     name = Aes128,
398 | |     name_enc = Aes128Enc,
399 | |     name_dec = Aes128Dec,
...   |
402 | |     doc = "AES-128",
403 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:330:13
    |
330 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
331 | |                 crate::weak_key_test(&key.0)
332 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
396 | / define_aes_impl!(
397 | |     name = Aes128,
398 | |     name_enc = Aes128Enc,
399 | |     name_dec = Aes128Dec,
...   |
402 | |     doc = "AES-128",
403 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:105:13
    |
105 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
106 | |                 crate::weak_key_test(&key.0)
107 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
404 | / define_aes_impl!(
405 | |     name = Aes192,
406 | |     name_enc = Aes192Enc,
407 | |     name_dec = Aes192Dec,
...   |
410 | |     doc = "AES-192",
411 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:209:13
    |
209 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
210 | |                 crate::weak_key_test(&key.0)
211 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
404 | / define_aes_impl!(
405 | |     name = Aes192,
406 | |     name_enc = Aes192Enc,
407 | |     name_dec = Aes192Dec,
...   |
410 | |     doc = "AES-192",
411 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:330:13
    |
330 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
331 | |                 crate::weak_key_test(&key.0)
332 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
404 | / define_aes_impl!(
405 | |     name = Aes192,
406 | |     name_enc = Aes192Enc,
407 | |     name_dec = Aes192Dec,
...   |
410 | |     doc = "AES-192",
411 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:105:13
    |
105 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
106 | |                 crate::weak_key_test(&key.0)
107 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
412 | / define_aes_impl!(
413 | |     name = Aes256,
414 | |     name_enc = Aes256Enc,
415 | |     name_dec = Aes256Dec,
...   |
418 | |     doc = "AES-256",
419 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:209:13
    |
209 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
210 | |                 crate::weak_key_test(&key.0)
211 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
412 | / define_aes_impl!(
413 | |     name = Aes256,
414 | |     name_enc = Aes256Enc,
415 | |     name_dec = Aes256Dec,
...   |
418 | |     doc = "AES-256",
419 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `weak_key_test` is not a member of trait `KeyInit`
   --> /home/mulder/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/aes-0.9.0-rc.2/src/autodetect.rs:330:13
    |
330 | /             fn weak_key_test(key: &Key<Self>) -> Result<(), WeakKeyError> {
331 | |                 crate::weak_key_test(&key.0)
332 | |             }
    | |_____________^ not a member of trait `KeyInit`
...
412 | / define_aes_impl!(
413 | |     name = Aes256,
414 | |     name_enc = Aes256Enc,
415 | |     name_dec = Aes256Dec,
...   |
418 | |     doc = "AES-256",
419 | | );
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `define_aes_impl` (in Nightly builds, run with -Z macro-backtrace for more info)

Some errors have detailed explanations: E0407, E0432.
For more information about an error, try `rustc --explain E0407`.
error: could not compile `aes` (lib) due to 31 previous errors
```

Since my project has dependency on `aes` v0.9.0-rc.2, any project using my project will now fail to build 😢 

Best regards.

--- a/aes/Cargo.toml
+++ b/aes/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "aes"
-version = "0.9.0-rc.2"
+version = "0.9.0-rc.3"
 description = "Pure Rust implementation of the Advanced Encryption Standard (a.k.a. Rijndael)"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/aria/Cargo.toml
+++ b/aria/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "aria"
-version = "0.2.0-rc.0"
+version = "0.2.0-rc.1"
 description = "Pure Rust implementation of the ARIA Encryption Algorithm"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/belt-block/Cargo.toml
+++ b/belt-block/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "belt-block"
-version = "0.2.0-rc.2"
+version = "0.2.0-rc.3"
 description = "belt-block block cipher implementation"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/blowfish/Cargo.toml
+++ b/blowfish/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "blowfish"
-version = "0.10.0-rc.2"
+version = "0.10.0-rc.3"
 description = "Blowfish block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/camellia/Cargo.toml
+++ b/camellia/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "camellia"
-version = "0.2.0-rc.0"
+version = "0.2.0-rc.1"
 description = "Camellia block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/cast5/Cargo.toml
+++ b/cast5/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "cast5"
-version = "0.12.0-rc.0"
+version = "0.12.0-rc.1"
 description = "CAST5 block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/cast6/Cargo.toml
+++ b/cast6/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "cast6"
-version = "0.2.0-rc.0"
+version = "0.2.0-rc.1"
 description = "CAST6 block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/des/Cargo.toml
+++ b/des/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "des"
-version = "0.9.0-rc.2"
+version = "0.9.0-rc.3"
 description = "DES and Triple DES (3DES, TDES) block ciphers implementation"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/gift/Cargo.toml
+++ b/gift/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "gift-cipher"
-version = "0.1.0-rc.2"
+version = "0.1.0-rc.3"
 description = "Pure Rust implementation of the Gift block cipher"
 authors = ["RustCrypto Developers", "Schmid7k"]
 license = "MIT OR Apache-2.0"

--- a/idea/Cargo.toml
+++ b/idea/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "idea"
-version = "0.6.0-rc.0"
+version = "0.6.0-rc.1"
 description = "IDEA block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/kuznyechik/Cargo.toml
+++ b/kuznyechik/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "kuznyechik"
-version = "0.9.0-rc.2"
+version = "0.9.0-rc.3"
 description = "Kuznyechik (GOST R 34.12-2015) block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/magma/Cargo.toml
+++ b/magma/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "magma"
-version = "0.10.0-rc.2"
+version = "0.10.0-rc.3"
 description = "Magma (GOST R 34.12-2015) block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/rc2/Cargo.toml
+++ b/rc2/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rc2"
-version = "0.9.0-rc.0"
+version = "0.9.0-rc.1"
 description = "RC2 block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/rc5/Cargo.toml
+++ b/rc5/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rc5"
-version = "0.1.0-rc.0"
+version = "0.1.0-rc.1"
 description = "RC5 block cipher"
 authors = ["RustCrypto Developers"]
 edition = "2024"

--- a/rc6/Cargo.toml
+++ b/rc6/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rc6"
-version = "0.1.0-rc.2"
+version = "0.1.0-rc.3"
 description = "RC6 block cipher"
 authors = ["RustCrypto Developers"]
 edition = "2024"

--- a/serpent/Cargo.toml
+++ b/serpent/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "serpent"
-version = "0.6.0-rc.2"
+version = "0.6.0-rc.3"
 description = "Serpent block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/sm4/Cargo.toml
+++ b/sm4/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "sm4"
-version = "0.6.0-rc.2"
+version = "0.6.0-rc.3"
 description = "SM4 block cipher algorithm"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/speck/Cargo.toml
+++ b/speck/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "speck-cipher"
-version = "0.1.0-rc.2"
+version = "0.1.0-rc.3"
 authors = ["RustCrypto Developers"]
 license = "Apache-2.0 OR MIT"
 description = "Speck block cipher algorithm"

--- a/threefish/Cargo.toml
+++ b/threefish/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "threefish"
-version = "0.6.0-rc.0"
+version = "0.6.0-rc.1"
 description = "Threefish block cipher"
 authors = ["The Rust-Crypto Project Developers"]
 license = "MIT OR Apache-2.0"

--- a/twofish/Cargo.toml
+++ b/twofish/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "twofish"
-version = "0.8.0-rc.0"
+version = "0.8.0-rc.1"
 description = "Twofish block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

--- a/xtea/Cargo.toml
+++ b/xtea/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "xtea"
-version = "0.1.0-rc.0"
+version = "0.1.0-rc.1"
 description = "XTEA block cipher"
 authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"

